packages/devcmd-cli/src/index.ts

```diff
@@ -58,7 +58,7 @@ async function startProcess(command: string, args: Array<string>, dirPath: strin
     const processInstance: ChildProcess = spawn(command, args, spawnOptions);
 
     processInstance.on("error", (err: Error): void => {
-      throw new Error(err.message);
+      throw err;
```

Comment:
Note that this will not be caught by the `catch` clause in line 85, as it's no real exception of the `Promise` (which would have to be reported by calling `reject`)

Reply:
Good catch! I just found out that the 'error' event is only emitted, when the process couldnt be started / stopped / communicated with. Ill reject the promise (as it was before) rather than throwing the error and document the error event.